Modules/Sources/AsyncImageKit/Views/AsyncImageView.swift

```diff
@@ -185,7 +185,7 @@ extension GIFImageView {
         }
     }
 
-    public func prepareForReuse() {
+    public func reset() {
```

Comment:
I moved this code to an extension without realizing the method named `prepareForReuse` already existed on `GIFAnimatable`. There needs to be a more durable solution, but I fixed it like this for now.

The issue with the standard `prepareForReuse` is that it only works for animated images, AFAIR.